src/utils/intersection-observer-polyfill.js

```diff
@@ -14,9 +14,6 @@
  * limitations under the License.
  */
 
-// TODO(#27807): Remove polyfill part InOb polyfill has been launched. But the
```

Comment:
I think it's worth changing the `IntersectionObserverHostApi`, `IntersectionObserverHostForAd` 
 class later on, also the file name to be more clear : )

Reply:
Sure. But I think it still covers more than ads. It's more of a "legacy 3p intersection observer host support".